tools/run_tests/xds_k8s_test_driver/bin/cleanup/cleanup.py

```diff
@@ -58,6 +58,12 @@
 PSM_SECURITY_PREFIX = 'xds-k8s-security'  # Prefix for gke resources to delete.
 URL_MAP_TEST_PREFIX = 'interop-psm-url-map'  # Prefix for url-map test resources to delete.
 
+KEEP_PERIOD_DAYS = flags.DEFINE_integer(
+    "keep_days",
```

Comment:
I would use seconds or hours... In the worst case, we might need to remove resources that are several hours old.

Reply:
Done